### ISSUE TITLE
jplhorizons remove deprecated columns in the code

### ISSUE THREAD

**Issue by ysBach** [astropy/astroquery#2913]
There are a few deprecated columns that are no longer being queried, but remaining in the ``__init__.py``:

https://github.com/astropy/astroquery/blob/9387c450137dd6ed0f9c2c96df4abfcf201059a8/astroquery/jplhorizons/__init__.py#L104-L113

I think it is safe to remove those lines because the HORIZONS no longer returns such columns...?

**Comment by bsipocz** [comment]
cc @mkelley

**Comment by bsipocz** [comment]
I suppose these are not required but are still listed to ensure old code using them don't yet start failing, but @mkelley can correct me and point out if it's time for a cleanup.

**Comment by mkelley** [comment]
Yeah, they should get removed.  I think the only drawback is that older cached files would fail.  When was the change made?  It might also require an update of the offline test files, depending on when those files where generated and when Horizons changed its output.
